File: app/src/main/java/game/ivan/ashancalculator/items/view/ItemsView.java
package game.ivan.ashancalculator.items.view;

import com.hannesdorfmann.mosby.mvp.MvpView;

import java.util.List;

import game.ivan.ashancalculator.database.models.Item;
import io.reactivex.Observable;

/**
 * Created by ivan on 21.12.16.
 */

public interface ItemsView extends MvpView {

    void refreshView(Observable<List<Item>> list);

    void setImagePath(String path);

    void showEditDialog(Item item);

    void showCreateDialog();
}

File: app/src/main/java/game/ivan/ashancalculator/items/view/ItemsView.kt
package game.ivan.ashancalculator.items.view

import com.hannesdorfmann.mosby.mvp.MvpView

import game.ivan.ashancalculator.database.models.Item

/**
 * Created by ivan on 21.12.16.
 */

interface ItemsView : MvpView {

    fun refreshView(list: List<Item>)

    fun setImagePath(path: String)

    fun showEditDialog(item: Item)

    fun showCreateDialog()
}
